### ISSUE TITLE
State 'virt.keys' was not found in SLS 'libvirt.keys' for Salt 2017.7.3.0

### ISSUE THREAD

**Issue by stooj** [saltstack-formulas/libvirt-formula#29]
```
----------
          ID: libvirt.keys
    Function: virt.keys
        Name: libvirt_keys
      Result: False
     Comment: State 'virt.keys' was not found in SLS 'libvirt.keys'
              Reason: 'virt' __virtual__ returned False
     Changes: 
```

My minion is running Ubuntu 17.10, and this is failing because the `python-libvirt` package has not been installed.

My top file simply includes `libvirt`, and `extra_pkgs` returns correctly.

```
----------
          ID: extra_pkgs
    Function: pkg.installed
      Result: True
     Comment: All specified packages are already installed
     Started: 22:51:39.869064
    Duration: 5.806 ms
     Changes:
```

**Comment by mtlott** [comment]
This is still an issue in 2018.3.0 (Oxygen)

**Comment by baby-gnu** [comment]
I finish the fix in #47, when it will be merged I'll start to pass on all distributions.

**Comment by baby-gnu** [comment]
I'm closing since the initial issue was for Ubuntu and it's now fully supported.

Thanks a lot.
